airbyte-ci/connectors/metadata_service/lib/metadata_service/validators/metadata_validator.py

```diff
@@ -64,9 +76,43 @@ def validate_all_tags_are_keyvalue_pairs(metadata_definition: ConnectorMetadataD
     return True, None
 
 
+def is_major_version(version: str) -> bool:
```

Comment:
Just leaving a note here from our call.

This does open us up to a cheeky dev who goes from `3.9.9` to `4.0.1` to avoid the check.

Not likely

but it does make me look forward to the version that references the latest remote metadata :) 

Do we have a tech debt issue for that @erohmensing

Reply:
https://app.zenhub.com/workspaces/connector-operations-6318ede5eda32e0010ef8566/issues/gh/airbytehq/airbyte/28412